save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityGeneralInfo.kt

```diff
@@ -56,13 +57,16 @@ val vulnerabilityGeneralInfo: FC<VulnerabilityGeneralInfo> = FC { props ->
                         }
                     }
                     if (isEditDisabled) {
-                        buttonBuilder(
-                            labelBuilder = {
-                                +"Edit "
-                                fontAwesomeIcon(icon = faEdit)
-                            },
-                            "link", isOutline = true, classes = "text-xs text-muted text-left ml-auto") {
-                            setIsEditDisabled(false)
+                        if (props.userInfo?.isSuperAdmin() == true || props.userInfo?.name == userInfo.name) {
```

Comment:
It also needs a check for the _isEditDisabled_ flag.
And your **If** must be common to the entire _if-else_ block